include/eve/module/core/decorator/promote.hpp

```diff
@@ -0,0 +1,57 @@
+//==================================================================================================
+/*
+  EVE - Expressive Vector Engine
+  Copyright : EVE Project Contributors
+  SPDX-License-Identifier: BSL-1.0
+*/
+//==================================================================================================
+#pragma once
+
+#include <eve/detail/overload.hpp>
+
+namespace eve
+{
+template<auto Param> struct diff_;
```

Comment:
Stop with the copy paste.

Reply:
I thought that pedantic was ok
I had not seen diff